src/pages/admin/index.tsx

```diff
@@ -1,91 +1,141 @@
 import React from 'react'
 import styled from 'styled-components'
 import { navigate } from 'gatsby'
-
+import {
+    AiFillFileAdd,
+    FaUserFriends,
+    FcEnteringHeavenAlive,
+    SiGoogletagmanager,
+} from 'react-icons/all'
+import { IconType } from 'react-icons/lib'
 import { AdminLayout } from '../../shared/components'
+import LandingButton from '../../shared/components/Button/link'
+
+type Props = {
+    text: string
+    color: string
+    icon: IconType
+    redirection: () => void
+}
 
 const Container = styled.div`
-    display: flex;
-    justify-content: flex-start;
-    align-items: center;
-    gap: 25px;
+    background-color: #f9f9f9;
+    padding: 100px;
 `
 
-const CardContainer = styled.div`
-    border: black 1px solid;
-    border-radius: 15px;
+const Wrapper = styled.div`
     display: flex;
-    justify-content: center;
-    align-items: center;
-    cursor: pointer;
-    padding: 10px;
 `
 
-const Title = styled.h1`
-    font-size: 1.3em;
+const LandingButtonWrapper = styled.div`
+    width: 100%;
 `
 
-const Icon = styled.img`
-    height: 50px;
+const TextContainer = styled.div`
+    flex: 1;
+    padding-right: 50px;
 `
 
-type Card = {
-    title: string
-    onClick: () => void
-}
+const TitleWrapper = styled.div`
+    margin-bottom: 50px;
+`
 
-const Dashboard = () => {
-    const cards: Array<Card> = [
-        {
-            title: 'Create content',
-            onClick: () => {
-                navigate('/admin/create')
-            },
-        },
-        {
-            title: 'Manage Content',
-            onClick: () => {
-                navigate('/admin/manage')
-            },
-        },
-        {
-            title: 'Create Sections',
-            onClick: () => {
-                navigate('/admin/sections/create')
-            },
-        },
-        {
-            title: 'Manage Sections',
-            onClick: () => {
-                navigate('/admin/sections/manage')
-            },
-        },
-        {
-            title: 'Livestream',
-            onClick: () => {
-                navigate('/admin/livestream')
-            },
-        },
-        {
-            title: 'Submissions management',
-            onClick: () => {
-                navigate('/admin/submissions')
-            },
-        },
-    ]
+const Title = styled.p`
+    color: ${(props) => props.color || '#000000'};
+    font-size: 48px;
```

Comment:
Decrease the text size
```suggestion
    font-size: 38px;
```